### ISSUE TITLE
model.fit()---the accuracy on the test set remains the same

### ISSUE THREAD

**Issue by Eurus-Holmes** [keras-team/keras#10996]
When I test the unbalanced sample label, the two-category label is divided by about 6:4, but val_acc remains the same.

Here is a similar [issue: ](https://stackoverflow.com/questions/47710566/class-weight-auto-for-model-fit-generator-keras)

But it doesn't work for me...

I think it is because keras has set up an anti-unbalance training mechanism?

Did anyone else face this problem? How can I save it...

<img width="518" alt="2018-08-26 9 21 03" src="https://user-images.githubusercontent.com/34226570/44624459-e7a27a00-a921-11e8-97a0-87582aab1105.png">
